### PR TITLE
feat(tier4_system_launch): remove configs and move to autoware_launch

### PR DESCRIPTION
**Pull request by kminoda**
Signed-off-by: kminoda <koji.minoda@tier4.jp>

## Description
Move config to autoware_launch.

Two PRs should be merged at the same time.
- https://github.com/autowarefoundation/autoware.universe/pull/2540
- https://github.com/autowarefoundation/autoware_launch/pull/130

## Related links
https://github.com/autowarefoundation/autoware_launch/issues/66

## Tests performed
Launch logging_simulator.launch.xml and verified that all the nodes in `/system/*` are launched successfully.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_system_launch/README.md [hunk 1]
```diff
@@ -10,10 +10,14 @@ Please see `<exec_depend>` in `package.xml`.
...
 ## Usage
 
+Note that you should provide parameter paths as `PACKAGE_param_path`. The list of parameter paths you should provide is written at the top of `perception.launch.xml`.
+
 ```xml
   <include file="$(find-pkg-share tier4_system_launch)/launch/system.launch.xml">
     <arg name="run_mode" value="online"/>
     <arg name="sensor_model" value="SENSOR_MODEL"/>
+
+    <arg name="PACKAGE_param_path" value="..."/>
```
[the remark this answers is not in the document]
Comment (kenji-miyake) [reply launch/tier4_system_launch/README.md:20]: Also, since you added `  <!-- Parameter files -->` and wrote parameters right under the `<launch>` tag, I think it's good to unify the formats of this example and the actual launch file.

File: launch/tier4_system_launch/launch/system.launch.xml [hunk 2]
```diff
@@ -13,15 +29,15 @@
...
-        <arg name="cpu_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/cpu_monitor.param.yaml"/>
-        <arg name="hdd_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/hdd_monitor.param.yaml"/>
-        <arg name="mem_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/mem_monitor.param.yaml"/>
-        <arg name="net_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/net_monitor.param.yaml"/>
-        <arg name="ntp_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/ntp_monitor.param.yaml"/>
-        <arg name="process_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/process_monitor.param.yaml"/>
-        <arg name="gpu_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/gpu_monitor.param.yaml"/>
-        <arg name="voltage_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/voltage_monitor.param.yaml"/>
+      <include file="$(find-pkg-share system_monitor)/launch/system_monitor.launch.py">
```
Comment (kenji-miyake) [inline launch/tier4_system_launch/launch/system.launch.xml:32]: Why did you change here from `.xml` to `.py`?

File: launch/tier4_system_launch/README.md [hunk 3]
```diff
@@ -10,10 +10,14 @@ Please see `<exec_depend>` in `package.xml`.
 
 ## Usage
 
+Note that you should provide parameter paths as `PACKAGE_param_path`. The list of parameter paths you should provide is written at the top of `perception.launch.xml`.
```
[the remark this answers is not in the document]
Comment (kenji-miyake) [reply launch/tier4_system_launch/README.md]: Renamed to `map` in 86b6ab0a0212d8d833d6c863c70a01cea2a21046, but still seems to be wrong.

File: launch/tier4_system_launch/CMakeLists.txt [hunk 4]
```diff
@@ -6,5 +6,4 @@ autoware_package()
 
 ament_auto_package(INSTALL_TO_SHARE
   launch
-  config
-)
+  )
```
[the remark this answers is not in the document]
Comment (kminoda) [reply launch/tier4_system_launch/CMakeLists.txt]: Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2540/commits/7b1e706fcc4254b863a00b03ba085f9fbb8adec0

File: launch/tier4_system_launch/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kminoda) [reply launch/tier4_system_launch/README.md]: Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2540/commits/57a3b7524b292c64f7fd4569506237994fdda583

File: launch/tier4_system_launch/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kminoda) [reply launch/tier4_system_launch/README.md]: Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2540/commits/57a3b7524b292c64f7fd4569506237994fdda583

File: launch/tier4_system_launch/launch/system.launch.xml (hunk 2 above)
Comment (kminoda) [reply launch/tier4_system_launch/launch/system.launch.xml]: Thank you for all the review! (and sorry for being late)
Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2540/commits/767a0854fa00eeb8e1ab43d01c8473e1e822e2ff

**Review by kenji-miyake — APPROVED** [review]
Cool! It's simplified!

**Comment by kenji-miyake** [comment]
@kminoda Could you fill the `Tests performed` section, please?
